src/Custom.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+
+  <PropertyGroup>
+	<LangVersion>9.0</LangVersion>
```

Comment:
Otherwise you have a fight between NET6 analyzers wanting `(e is not AssertionException)` above, and net452 using older C# and not having a clue what that means. In an all-tests project targeting many frameworks, just setting the LangVersion globally seemed preferable to an `.editorconfig` file in every single project directory.